src/explore-education-statistics-admin/src/components/comments/CommentsWrapper.tsx

```diff
@@ -0,0 +1,101 @@
+import styles from '@admin/components/comments/CommentsWrapper.module.scss';
+import CommentAddForm from '@admin/components/comments/CommentAddForm';
+import CommentsList from '@admin/components/comments/CommentsList';
+import { useCommentsContext } from '@admin/contexts/CommentsContext';
+import React, { ReactNode, useRef } from 'react';
+import { CommentType } from 'src/services/types/content';
+import classNames from 'classnames';
+import { Toggle } from '@common/hooks/useToggle';
+import Button from '@common/components/Button';
+import Tooltip from '@common/components/Tooltip';
+
+interface Props {
+  allowComments?: boolean;
+  blockLockedMessage?: string;
+  children: ReactNode;
+  commentType: CommentType;
+  id: string;
+  showCommentAddForm?: boolean;
+  showCommentsList?: boolean;
+  testId?: string;
+  toggleCommentAddForm?: Toggle;
+  onViewComments?: () => void;
+}
+
+export default function CommentsWrapper({
+  allowComments = true,
+  blockLockedMessage,
+  children,
+  id,
+  commentType,
+  showCommentAddForm,
+  showCommentsList = true,
+  testId,
+  toggleCommentAddForm,
+  onViewComments,
+}: Props) {
+  const containerRef = useRef<HTMLDivElement>(null);
+  const { comments } = useCommentsContext();
+  const isInline = commentType === 'inline';
+
+  return (
+    <div className={styles.container} ref={containerRef} data-testid={testId}>
+      {allowComments && (
+        <div className={styles.sidebar} data-testid="comments-sidebar">
```

Comment:
I demo'd this to Cam yesterday and he's ok with having it as it is, which admittedly isn't ideal but it is useable which is an improvement on how it is currently. He's aware that in general the mobile / tablet layout in the admin needs to be reviewed.

It'll definitely need some more thought when we do look at improving it:
- If we moved the comments to below the block on mobile then on desktop we'd have to move them to be on the right otherwise the tab order would be confusing.
- I don't think having the comments list open in a modal would work as for inline comments you wouldn't be able to see what the comments refer to.